test/data/testSP.py

```diff
@@ -241,19 +241,10 @@ def testmoments(self):
                 self.assertEquals(m, 0.0)
 
     @skipForParser('ADF', 'Does not support metadata yet')
-    @skipForParser('GAMESS', 'Does not support metadata yet')
     @skipForParser('GAMESSUK', 'Does not support metadata yet')
-    @skipForParser('Gaussian', 'Does not support metadata yet')
-    @skipForParser('Jaguar', 'Does not support metadata yet')
-    @skipForParser('Molpro', 'Does not support metadata yet')
-    @skipForParser('NWChem', 'Does not support metadata yet')
-    @skipForParser('ORCA', 'Does not support metadata yet')
-    @skipForParser('Psi', 'Does not support metadata yet')
-    @skipForParser('QChem', 'Does not support metadata yet')
     def testmetadata(self):
         """Does metadata have expected keys and values?"""
         self.assertTrue(hasattr(self.data, "metadata"))
-        self.assertIn("basis_set", self.data.metadata)
         self.assertIn("methods", self.data.metadata)
         self.assertIn("package", self.data.metadata)
         self.assertIn("package_version", self.data.metadata)

```

Comment:
Why was ```testSP.py``` changed for this pull request? Seems like it's mostly unrelated, and should be a separate PR.

Reply:
I don't think we will every standardize `metadata` since the whole point was to parse things in there that are useful but are not necessarily viable attributes. But that doesn't mean we can't test it and expect specific contents for specific cases.

I think we should leave this is in the test. I think we should expect it to be parsed at least for out unit test suite.

Reply:
In ORCA there is simply no easy way to parse a basis set name without having a list of all possible basis set names, since they are passed in using the multikeyword bang line. I have access to the ORCA source code and thus could find all the possible  basis sets (~400), but this would be rather ugly and subject to change. Instead, we could parse the basis set information block.

```
BASIS SET INFORMATION
---------------------
There are 3 groups of distinct atoms

 Group   1 Type Ir  : 7s6p5d1f contracted to 6s3p2d1f pattern {211111/411/41/1}
 Group   2 Type O   : 7s4p1d contracted to 3s2p1d pattern {511/31/1}
 Group   3 Type H   : 4s1p contracted to 2s1p pattern {31/1}

Atom   0Ir   basis set group =>   1
Atom   1Ir   basis set group =>   1
Atom   2O    basis set group =>   2
```

Would this be useful? I'm not sure what there is of value in this section.
How should I proceed?